js/src/external-components/wordpress/guide/index.js

```diff
@@ -143,7 +144,10 @@ export default function Guide( {
 							{ __( 'Next' ) }
 						</Button>
 					) }
+
 					{ finishBlock }
+
+					{ pages[ currentPage ]?.footer }
```

Comment:
Looks like the render props `renderFinish` could archive the same result without adding `footer`. Is there any reason to add it?

Reply:
@eason9487 I felt it'll be better if each screen has its content and footer defined in a single place (the `pages`) array compared to having the logic to display the footer in a separate function which feels like disconnected. What do you think? It's easier to read through the code knowing immediately what's the content and footer for each page.

Reply:
@eason9487 I felt it'll be better if each screen has its content and footer defined in a single place (the `pages` array) compared to having the logic to display the footer in a separate function which feels like disconnected. What do you think? It's easier to read through the code knowing immediately what's the content and footer for each page.